crates/amaru/src/bin/amaru/metrics.rs

```diff
@@ -92,55 +62,47 @@ mod internals {
             .with_unit("MB")
             .build();
 
+        #[cfg(not(windows))]
         let cpu_load = meter
             .f64_gauge("cpu.utilization")
             .with_description("the 1m average load, measured once per second")
             .build();
 
-        let user_time = meter
-            .u64_counter("cpu.time")
-            .with_description("the total cpu time spent in user processes")
+        let cpu_usage = meter
+            .f64_counter("cpu.percent")
+            .with_description("the cpu usage in percent, measured once per second")
             .with_unit("ms")
```

Comment:
```suggestion
            .with_unit("%")
```

Seems like the number we're getting can greatly go above 100% though. The documentation says the following:

> Notice that it might be bigger than 100 if run on a multi-core machine. If you want a value between 0% and 100%, divide the returned value by the number of CPUs. 

Although, I only have 9 CPUs; and so even if considering CPU's threads, I don't see how numbers above 3000 can make sense 😅 (especially when Amaru is mostly idle):

<img width="1495" alt="image" src="https://github.com/user-attachments/assets/6f29f130-5976-4736-9960-ab2983ed85ab" />